doc/enterprise/audit.rst

```diff
@@ -0,0 +1,185 @@
+Security audit
+==============
+
+This document will help you audit the security of a Tarantool Enterprise Edition's cluster.
```

Comment:
Possessive looks incorrect here. If `Tarantool Enterprise Edition` is a product name, then it should be in the simple form, just as `Tarantool cluster`, `GridGain cluster` (not `Tarantool's cluster`).